src/main/java/com/arabiccalligraphy/api/DAO/ArtworkDAO.java

```diff
@@ -11,41 +12,65 @@
 import java.io.File;
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 @Service
 public class ArtworkDAO {
-    private List<Artwork> artworks = new ArrayList<>();
+    private Map<String, List<Artwork>> artworks = new HashMap<>();
 
-    public List<Artwork> getArtworks() {
+    public Map<String, List<Artwork>> getArtworks() {
         return artworks;
     }
 
     @PostConstruct
     public void loadArtwork() throws IOException {
 
-        File file = ResourceUtils.getFile("classpath:data/index.json");
+        artworks.put("index.json", new ArrayList<>());
+        artworks.put("kamel.json", new ArrayList<>());
+        artworks.put("mokhtar.json", new ArrayList<>());
+
+        loadArtworkFromFile("index.json");
+        loadArtworkFromFile("kamel.json");
+        loadArtworkFromFile("mokhtar.json");
+    }
+
+
+    private void loadArtworkFromFile(String fileName) throws IOException {
+
+        File file = ResourceUtils.getFile("classpath:data/" + fileName);
 
         ObjectMapper mapper = new ObjectMapper();
         JsonNode root = mapper.readTree(file);
 
         ArrayNode contentNode = (ArrayNode) root.get("content");
         for (JsonNode contentItem : contentNode) {
+
+            //title creation
             String title = contentItem.get("nameAr").asText();
             if (title.equals("null")) {
                 title = contentItem.get("name").asText();
             }
+
+            //image creation
             String image = contentItem.get("imgSrc").asText();
 
+            //tags creation
             ArrayNode tagsNode = (ArrayNode) contentItem.get("tags");
             String[] tags = new String[tagsNode.size()];
-
             for (int i = 0; i < tagsNode.size(); i++) {
                 tags[i] = tagsNode.get(i).asText();
             }
 
-            Artwork artwork = new Artwork(title, tags, image, null);
-            artworks.add(artwork);
+            //Artist creation
+            Artist artist = new Artist(contentItem.get("artist").asText());
```

Comment:
Maybe in the future, we can add more fields to the artist, but his looks good for now.